osu.Game.Rulesets.Osu/Skinning/Legacy/LegacySpinner.cs

```diff
@@ -101,8 +104,28 @@ private void load(DrawableHitObject drawableHitObject, ISkinSource source)
                     }.With(s => s.Font = s.Font.With(fixedWidth: false)),
                 }
             });
+
+            static Drawable getSpinnerApproachCircle(ISkinSource source)
+            {
+                var spinnerProvider = source.FindProvider(s =>
+                    s.GetTexture("spinner-circle") != null ||
+                    s.GetTexture("spinner-top") != null);
+
+                if (spinnerProvider is DefaultLegacySkin)
+                    return Empty();
```

Comment:
This may not be the right way to check if the `spinnerProvider` is coming from the fallback default skin (to behave just like the [wiki](https://osu.ppy.sh/wiki/en/Skinning/osu%21#spinner) states), since there could be modified skins based off the osu!classic skin.

Unless modifying default skins will not allow access to changing the textures they provide, this may have to change to something like `spinnerProvider == skinManager.DefaultLegacySkin` to be more correct, maybe?